From 5ddd8baa4857709b4e5d84b376d735152851955b Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Google)" <rostedt@goodmis.org>
Date: Thu, 5 Oct 2023 10:47:45 -0400
Subject: tracing: Make system_callback() function static

From: Steven Rostedt (Google) <rostedt@goodmis.org>

commit 5ddd8baa4857709b4e5d84b376d735152851955b upstream.

The system_callback() function in trace_events.c is only used within that
file. The "static" annotation was missed.

Fixes: 5790b1fb3d672 ("eventfs: Remove eventfs_file and just use eventfs_inode")
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202310051743.y9EobbUr-lkp@intel.com/
Signed-off-by: Steven Rostedt (Google) <rostedt@goodmis.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 kernel/trace/trace_events.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/trace/trace_events.c
+++ b/kernel/trace/trace_events.c
@@ -2302,7 +2302,7 @@ create_new_subsystem(const char *name)
 	return NULL;
 }
 
-int system_callback(const char *name, umode_t *mode, void **data,
+static int system_callback(const char *name, umode_t *mode, void **data,
 		    const struct file_operations **fops)
 {
 	if (strcmp(name, "filter") == 0)
